core/assets/bundles/bundle_de.properties

```diff
@@ -965,7 +965,7 @@ ability.statusfield = {0} Statusfeld
 ability.unitspawn = {0} Fabrik
 ability.shieldregenfield = Schild-regenerations-Feld
 ability.movelightning = Bewegungsblitze
-ability.shieldarc = Lichtbogenschild
+ability.shieldarc = Bogenschild
```

Comment:
If you're really going to change it, you should also change the other translations of "arc".

Also, shouldn't it be "Schild(licht)bogen" anyway? The original word is shield arc, not arc shield.